innopoints/core/image.py

```diff
@@ -0,0 +1,47 @@
+"""Image manipulation."""
+
+from typing import Dict
+
+from PIL import Image
+
+from innopoints.core.helpers import abort
+
+
+def crop(image: Image.Image, dimensions: Dict[str, str]):
+    """Crop an image to the dimensions specified with keys 'x', 'y', 'width', 'height'."""
+    if 'x' not in dimensions:
+        return image
+
+    try:
+        # pylint: disable=invalid-name
+        x = int(dimensions['x'])
+        y = int(dimensions['y'])
+        width = int(dimensions['width'])
+        height = int(dimensions['height'])
+    except KeyError:
+        abort(400, {'message': 'Not enough data to perform the crop.'})
```

Comment:
Should you really use `abort` from within a utility function? Wouldn't it be better to return `None` - for example - and check for it in the route function?

Reply:
I thought about it, but it was ugly either way. Abort throws exceptions, so it's okay

Reply:
I don't think that returning `None` or throwing a `ValueError` or something can be uglier than using `abort` outside a route function